pages/education/opt-out-information-sharing/opt-out-form-0993.md

```diff
@@ -1,15 +0,0 @@
----
-title: Opt Out Of Sharing VA Education Benefits Information
-heading: Opt out of sharing VA education benefits information
-entryname: 0993-edu-benefits
-layout: page-react.html
-description: Use your VA education benefits to pay for college or training programs. Find out which documents you’ll need to apply for benefits, and start your online application today.
----
-<nav aria-label="Breadcrumb" aria-live="polite" class="va-nav-breadcrumbs"
-id="va-breadcrumbs">
-  <ul class="row va-nav-breadcrumbs-list columns" id="va-breadcrumbs-list">
-    <li><a href="/">Home</a></li>
-    <li><a href="/education/">Education</a></li>
-    <li><a aria-current="page" href="/education/opt-out-information-sharing/opt-out-form-0993">Opt out of sharing your information with schools</a></li>
-  </ul>
-</nav>
```

Comment:
Migrated here, https://github.com/department-of-veterans-affairs/vets-website/pull/10660/files#diff-76623265ddc804707e763a34be07b7d7R5.